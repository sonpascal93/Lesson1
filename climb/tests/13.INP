5000
234 -217 160 277 763 827 260 832 -212 5 -561 961 -990 30 0 -465 142 566 -464 260 499 -845 -3 289 -643 611 58 -604 -102 -566 789 468 881 -672 -732 92 -245 318 -625 470 -510 -49 947 299 -252 929 245 -549 -453 221 789 -429 504 -180 93 525 -243 -928 606 -656 -687 -357 -808 -661 -647 46 330 -832 -501 -585 88 208 -997 374 816 578 -144 -148 325 -356 -514 500 -460 619 -251 132 394 -406 -946 -429 772 337 -918 -973 52 -18 355 381 691 -711 -99 636 -738 -231 -59 883 -535 -804 98 791 -887 -864 805 171 -942 -165 -573 595 416 3 -327 -210 -348 -917 -723 -940 -132 -708 214 -101 238 331 528 792 -973 266 -231 -864 89 -675 -930 -275 499 -664 -726 -82 546 395 -845 -387 289 4 -578 -983 510 -982 110 -868 37 -746 -510 780 243 -371 -427 -292 419 -961 -433 454 77 -502 376 471 -41 -732 -758 -2 272 -645 899 403 524 -481 347 -845 587 36 704 -52 376 -301 -685 76 -826 -664 748 -760 -475 276 -433 -940 -786 446 -253 796 82 -428 532 -778 -433 475 647 691 -249 122 -59 -507 754 197 512 -101 838 -923 -562 -332 -796 192 -364 669 230 396 384 226 283 818 -992 -816 501 193 64 -410 260 -758 386 -916 209 0 824 -297 -36 435 462 774 -566 -246 582 -436 40 -253 -993 -106 386 280 821 832 -853 -777 -945 253 -634 154 -451 -282 297 -285 258 -125 67 -802 -605 -695 143 -637 794 982 -249 -821 31 -500 -62 830 -277 -78 497 576 664 663 330 -638 -312 674 594 -751 939 -627 -544 -928 865 -750 799 -600 -511 30 -869 -771 133 720 261 48 351 413 -286 -987 -167 -908 -267 780 913 237 465 -326 -910 -263 715 881 930 -60 313 -712 -583 303 -69 315 -735 -13 389 190 789 129 828 480 133 750 217 173 -536 -601 14 -773 257 228 626 619 -938 -357 -879 214 672 -349 69 -275 474 217 853 -786 -622 740 263 -165 852 203 789 748 50 -703 510 -755 870 -15 676 818 463 388 -565 980 -748 -907 -886 -112 259 342 -659 411 452 -539 -48 -933 -667 -691 -436 -280 -866 -621 -369 -172 -388 313 205 413 507 853 -340 -27 683 387 -651 417 -5 481 -150 -318 477 808 -779 992 -580 933 -165 596 447 -427 163 39 -221 839 -942 -616 125 770 -654 979 470 739 942 432 686 747 99 -394 -827 31 705 -425 -421 593 278 40 -770 447 565 -695 -920 427 924 794 -927 537 -966 169 548 -884 -434 -670 -96 914 802 610 -158 -603 45 -688 648 -126 966 872 -107 467 -851 -197 319 -978 294 535 200 -787 -743 50 375 -708 -48 -694 98 -847 896 -549 -103 696 -857 128 -101 -11 474 226 -284 716 -965 555 -168 278 -346 -486 -137 341 946 -107 -478 -219 -565 -390 574 -495 -644 743 -829 -697 -848 -600 365 863 -426 562 -202 340 -520 -320 701 371 886 146 82 24 -227 315 -290 -200 -800 411 -634 284 -948 -87 -914 658 -131 -484 951 628 -178 887 -137 735 -981 944 -462 -644 -401 130 485 -811 762 541 483 91 -806 -173 690 835 -984 -440 -861 229 303 723 -173 -805 -732 702 -751 447 391 859 -988 550 -306 -387 -799 -829 484 -969 -67 -834 -240 181 -618 710 -316 -234 -631 911 11 497 -502 150 187 -804 -974 -1 -287 -27 -437 -346 910 115 940 166 106 -154 -983 647 -290 764 -31 -394 -120 -657 352 -266 836 988 27 243 -8 270 -285 223 -545 -449 861 85 788 781 298 365 239 103 563 -822 -468 -125 -701 0 -650 -291 -345 826 -51 713 -468 -989 830 610 -709 495 -719 -63 554 -313 170 858 -691 -353 308 364 541 697 -34 423 880 -606 -328 28 -186 946 315 14 826 314 600 -941 -294 -497 379 589 -53 179 40 332 -501 131 -237 688 239 414 606 -790 990 -149 -568 988 -228 -991 -6 280 893 -852 -424 710 673 -963 414 318 -382 -557 946 332 -892 -496 -483 -382 7 773 307 674 -665 -26 754 997 -781 -913 496 165 -325 367 439 -376 821 -774 958 -538 -470 420 -528 -667 60 -785 611 -631 369 725 177 900 -31 184 -322 -632 -441 871 512 0 -43 -30 -322 170 813 440 946 428 -125 -81 -519 578 418 -557 123 732 -345 -414 838 590 548 -298 -865 -727 -150 -226 308 -602 264 -404 185 -967 -911 690 -470 -299 -411 -66 231 932 106 596 858 -759 99 790 465 -106 -868 -59 707 1 -341 727 -443 781 -90 845 -208 -168 -605 538 300 -915 -209 703 445 -769 224 715 -846 -305 918 -919 -343 855 -947 -456 -969 -763 -877 -488 451 505 -44 -445 -424 -136 -197 -904 -350 -835 856 800 161 -547 -377 -261 902 179 -506 -13 -264 -506 -328 -106 958 -796 -314 506 -435 529 -237 317 -613 -125 -338 499 -167 -479 -430 580 -607 -954 -561 -798 -528 -343 240 170 345 -977 827 431 804 -951 488 -61 840 893 -630 -276 -280 916 -963 779 -193 733 518 -321 -720 882 901 503 -842 261 345 -292 -474 7 -469 88 -198 -227 -210 -593 -634 368 934 -305 -204 -75 249 -182 -534 746 151 489 -272 -122 496 -423 907 34 -571 -851 -695 -877 -214 -194 757 -556 888 -358 -734 914 992 436 75 638 -587 741 -186 -884 602 -560 615 -229 585 215 -544 698 337 470 -545 5 15 487 494 305 562 497 -220 -389 -169 -636 72 -868 427 -350 -365 -792 -214 121 -539 482 -273 -131 -554 -492 361 573 -91 183 184 45 224 -485 -729 124 648 -361 165 344 -316 -58 137 -498 -252 4 853 -41 522 174 967 -654 93 -948 462 -694 536 743 -264 796 -145 -521 869 276 -226 -778 -289 -88 483 -317 -413 727 654 731 -231 601 -698 -455 757 132 -17 48 -382 555 602 -443 754 202 -56 -574 -621 267 -16 198 -768 -979 679 -570 -423 -861 -409 -294 548 -326 -227 -657 507 288 379 902 244 -864 403 -295 -612 264 201 510 -348 -989 -834 -962 216 218 -938 624 481 -819 -325 703 -708 -770 -986 -258 497 598 -642 770 637 -983 486 -178 -569 -556 150 -410 960 -301 875 76 -828 241 -366 498 -914 -265 990 -54 385 307 62 96 -871 -528 388 -331 -733 222 588 -856 -214 727 291 498 -67 216 -812 -41 -42 803 222 -456 -627 -293 68 926 961 -945 868 -896 -543 -411 633 -589 25 229 -945 249 -182 190 439 -83 -141 710 -914 927 -229 700 92 916 -163 -824 -764 575 95 -97 -509 71 -991 -568 189 -407 -867 -643 -934 -335 360 -587 -296 -130 86 647 -189 697 -10 -510 -396 -447 924 -712 543 452 241 734 333 211 666 454 -568 -907 252 -440 879 993 422 668 -478 -940 222 -697 538 944 -604 -583 141 -404 493 -15 624 -124 642 645 -635 -608 278 589 447 -291 691 -83 -813 -28 -606 -641 788 680 -119 533 -624 326 -338 -504 309 -578 -744 -912 824 -451 415 -335 -638 -405 -717 434 -922 32 -193 523 -476 895 603 -552 -125 -177 303 -169 380 -339 -254 670 652 797 -820 871 -939 0 -131 -393 -308 775 -679 128 -326 123 640 471 -339 988 755 -58 -635 183 217 813 -70 -947 974 -410 733 995 507 -359 922 575 93 -509 603 -331 -540 -635 819 158 645 201 961 260 280 229 -975 -589 650 865 60 -700 -269 667 66 -777 -635 -363 313 55 -407 -497 -714 -430 -36 749 861 -74 -131 -932 -964 -2 217 -978 198 882 -359 795 -765 -98 -880 464 -503 -605 575 600 -377 562 383 877 -199 -543 400 667 389 -353 -609 -300 122 -813 376 894 735 322 -452 803 541 436 -516 -409 -28 -627 364 992 -797 -746 -335 -414 417 -977 232 718 -855 -563 -248 -765 226 880 388 -260 713 -52 972 -49 -205 54 324 565 -414 523 -935 -695 -471 440 639 736 461 -747 366 -61 -756 -915 -816 736 -130 443 399 -839 -353 468 869 340 42 -912 622 -771 365 983 -356 -839 -322 -821 256 -357 -643 186 -88 -368 863 -689 635 875 -273 -818 -614 698 950 -520 -147 554 -515 419 861 -636 -437 -359 22 822 -823 508 753 -585 152 700 968 -252 18 -592 -14 -640 219 -731 958 -274 -768 45 -491 220 933 -353 406 -957 -607 -423 -627 -92 -307 -515 644 113 -35 181 573 -76 -712 71 -174 619 379 538 804 456 -434 -185 304 -129 -425 -804 230 535 -543 -814 471 944 373 -320 -976 573 -210 116 417 -868 610 -634 254 502 -122 -302 594 -314 198 520 -759 -337 -955 -979 487 88 -741 455 695 26 -90 -928 -22 -331 941 -260 866 790 92 -48 811 999 539 -420 -623 673 423 -842 -100 -810 -795 -955 -265 -954 -686 658 -709 256 667 975 -134 693 -40 -101 -356 -783 592 -853 -179 71 977 513 756 267 662 -937 -349 -781 502 522 -413 -730 -107 -802 625 827 -844 149 553 584 -235 -955 -411 -384 240 823 -383 -579 452 -250 -874 896 -575 -398 -492 -279 -203 740 489 632 631 -790 -404 778 685 -477 -223 -861 -563 228 988 -872 326 -603 -775 -100 386 -270 401 -151 267 -59 759 -826 912 -439 -177 331 -3 458 484 884 -698 -478 561 810 -1 -940 -595 575 -829 360 61 -840 -274 -48 857 -992 95 696 858 -547 -165 -433 99 108 699 -221 -821 -799 -821 -412 157 -648 -744 803 -988 834 375 608 662 -873 -10 319 -739 57 -729 728 -673 -524 -525 -743 512 664 424 -442 701 302 -960 -783 815 444 729 235 -949 -44 424 -828 -174 -939 -744 -976 191 -275 -88 803 841 -473 955 5 320 -80 -62 -202 45 -548 -406 312 -61 888 -640 941 400 -641 951 -710 -146 -184 -242 537 -454 -596 427 156 -334 -453 201 587 -332 659 -134 -854 -813 -893 634 177 520 -866 390 811 -755 -963 -530 199 -502 918 -96 774 958 725 -854 -994 -84 661 -360 -412 627 160 564 600 -798 564 -68 636 -16 -346 855 -460 81 -431 -736 929 743 470 -574 -678 -363 -652 -154 -441 -440 474 765 -648 -936 -375 863 -625 223 -299 -545 -152 670 757 404 790 945 -439 655 -460 561 141 725 436 -906 -273 332 112 -142 -601 -968 113 -33 -855 166 -98 963 592 409 -58 -657 -927 664 -173 532 -191 174 -676 -464 788 -881 664 102 -42 -716 555 -667 316 -213 87 79 911 759 -341 791 213 25 560 -36 -786 -549 55 849 -178 522 -206 584 433 562 38 -822 626 -997 248 -770 569 -221 822 139 696 -544 -643 -335 341 369 -525 -156 202 -534 330 875 813 219 -416 -800 -806 -102 412 -179 436 -833 -932 650 335 -84 701 -709 231 136 462 48 -124 -733 -594 685 268 788 218 183 -632 -198 -625 -120 -911 739 -486 428 250 540 870 -980 534 -170 871 428 149 604 -430 749 -462 205 777 740 464 340 -214 -985 -450 858 -51 -313 -927 -672 -44 -492 170 -814 -528 -271 262 -52 -739 339 913 605 446 557 220 407 -227 -36 -308 557 -457 -406 892 -542 86 -11 -801 571 -576 -368 -429 -460 -35 94 597 -531 -654 248 -747 775 -566 -457 908 -555 97 515 242 -421 196 -633 -435 -471 -806 -158 -505 897 482 156 -545 765 961 -22 796 -986 430 -646 -512 -290 -896 -592 759 692 -276 -148 -703 -51 -952 704 137 -260 -129 684 891 -308 278 725 -550 -351 -219 -963 360 666 455 -98 835 755 446 -788 156 -948 135 -757 900 171 183 382 635 508 220 836 136 -622 899 -22 338 578 -947 -682 -819 -928 336 288 472 881 761 -37 -738 -422 377 241 151 -439 -330 745 -994 -965 446 743 -655 961 492 309 -88 354 -65 558 -58 -179 -732 -757 -560 570 649 -139 -24 -867 895 -125 313 259 978 108 -971 -85 665 -244 175 -672 377 893 563 591 512 373 -759 152 731 -820 482 -854 473 526 870 -839 -430 445 -776 -766 257 -572 985 -922 542 877 -704 -821 -100 484 -197 46 -904 -954 847 -432 873 -659 -999 97 -690 488 659 499 -91 486 -924 -910 -263 -444 -640 268 400 -665 -561 -916 -55 -652 588 -736 -284 -923 499 686 -572 -357 -539 -363 -781 -647 -343 -419 580 -626 -483 91 -656 211 -354 575 -421 762 841 -980 722 380 -233 -896 579 -922 -481 996 -480 560 601 113 540 686 360 720 -830 -970 -871 -200 427 379 -4 292 858 -663 -337 779 487 711 58 -895 9 6 -810 -134 332 558 -622 459 -32 -547 872 -926 888 -141 877 -152 764 874 -352 136 103 810 -653 -849 537 -954 735 313 198 -145 747 83 -429 -214 714 -286 -378 -512 -867 524 -740 188 -135 -109 8 -307 -43 -689 -173 993 -633 -73 398 -831 711 -655 -843 42 -262 -121 -440 74 588 -306 -996 364 523 625 -699 -424 -921 -48 181 636 758 798 -981 383 -192 981 -267 -806 -250 477 186 -507 608 -687 386 -41 245 -995 951 425 574 -403 -648 592 -893 -38 361 15 -406 -965 -154 -890 29 -977 734 185 -742 595 405 714 283 -659 -258 -981 626 -610 317 -374 -397 880 314 -671 -616 830 400 -491 166 565 -811 156 796 829 3 406 852 -15 -275 914 796 948 563 964 501 619 -254 317 428 -167 -476 921 -580 -567 -33 -970 170 964 420 -732 232 -326 -650 799 715 -111 -760 -416 748 895 68 -920 306 535 565 -652 519 -902 -430 427 63 254 784 643 740 232 -229 100 685 -260 -725 -371 -434 -130 92 584 865 364 -538 609 -388 -985 658 -637 -305 -76 -156 -562 569 738 195 375 -193 -848 -80 115 -972 -435 -637 -684 -383 94 250 384 -196 -96 435 232 -872 -291 -832 -752 -524 269 -972 -416 48 67 -695 790 803 209 804 74 636 612 52 -833 -989 352 609 -498 -410 154 -11 292 -326 -264 662 614 -207 443 -290 997 61 -884 554 -922 567 692 -622 -353 686 -779 -665 819 -519 -290 -4 -141 885 -544 881 -167 525 -585 -944 534 249 -754 -838 682 726 -295 -408 -648 -286 581 -565 -506 -814 216 -298 -882 600 294 35 523 659 -10 624 232 606 418 -341 -234 -873 842 -298 702 772 375 -177 844 901 396 67 -322 -103 -192 315 136 -126 -590 -929 -857 548 -684 553 480 718 -285 -985 -448 15 71 958 -555 -212 -765 8 700 -855 -14 -281 -285 357 452 -994 90 32 470 -502 -860 -453 78 645 -911 -375 564 -11 140 746 9 -542 -287 -57 141 83 674 920 114 -90 -121 771 943 61 644 -797 -14 605 -336 -978 781 901 646 365 -97 -653 -291 -618 -630 -49 -262 638 -160 831 325 -594 958 -221 448 -525 -194 -480 636 478 -50 -885 -409 -845 554 -397 597 459 -647 -441 -311 814 445 546 628 -358 731 -155 -294 211 -402 -77 520 572 697 490 420 401 764 -195 301 -18 -608 -202 917 -742 624 -480 -255 -381 -131 -664 -822 -366 772 -298 173 -362 -793 -506 765 -127 -477 815 986 86 -150 -860 982 -310 -432 -633 -556 -583 -272 -843 654 807 -161 352 384 -32 -522 268 -995 975 878 503 787 -247 -434 357 -121 668 -515 -501 -682 849 -494 218 142 627 -886 -348 -226 -979 -82 -24 494 816 -20 -428 -599 -314 419 -796 -31 -208 409 -905 -488 -11 118 -116 -892 265 27 439 -184 -188 -56 764 525 -880 503 740 982 632 -158 41 924 -321 -601 -437 973 981 805 -177 -512 -17 167 558 -371 -272 -155 -40 990 967 141 -853 989 350 244 155 673 213 -273 359 -880 117 773 647 281 98 -760 257 121 704 819 186 408 -739 -567 976 220 -261 779 -767 -477 -900 -138 512 -440 -898 618 589 444 851 502 839 -896 -90 -350 -137 -51 -236 -666 386 -346 -3 674 518 -766 789 409 -982 -900 131 -301 323 -267 650 609 407 -729 309 -123 957 -382 -473 -718 -499 551 895 393 727 -108 206 -406 471 35 -783 508 -488 -555 -39 -573 -320 889 344 721 -348 -655 -880 602 -306 968 145 781 -992 -109 -71 -991 -204 284 189 -558 17 -939 -864 -916 -241 -489 -563 -304 485 -358 -137 628 -46 589 286 -816 632 321 -103 -709 434 118 -420 307 47 -403 -21 -547 40 687 -779 -350 -688 649 451 -516 -110 862 15 430 591 374 -956 221 673 480 724 352 -281 -794 517 -978 -94 289 929 -722 -859 655 852 993 873 -664 -625 -192 339 956 -822 143 40 -689 891 638 -197 -422 -143 152 792 -960 738 643 -854 520 404 -2 206 -675 149 -739 547 874 939 322 254 -808 0 -588 -418 -734 -449 652 -446 354 -863 -761 151 467 -570 70 -611 743 329 -385 -223 29 427 828 378 -620 -178 383 847 -682 810 -604 -351 -790 862 -962 757 702 188 443 -57 -393 452 -86 845 327 653 773 491 228 648 424 -644 -1 -867 816 530 618 -727 -829 880 -625 -74 -833 181 -666 253 51 -573 733 -279 -372 751 -727 793 445 -824 -201 -728 -679 638 663 927 -994 -294 730 -922 711 -967 -247 -892 -174 381 -94 -257 -848 -911 -876 -133 -278 929 -313 -811 -608 794 891 -942 -359 -551 -673 748 -820 -414 680 -782 638 428 -130 874 176 -912 -466 617 -56 702 561 505 410 93 245 772 915 -809 -548 23 470 65 269 36 -426 34 -943 -280 -610 26 760 -446 -356 686 496 -386 955 133 619 767 -37 -579 -723 809 821 359 -443 132 -850 -509 -157 771 -430 -61 397 -594 632 -969 596 -277 83 -835 -534 -70 693 325 508 -738 -408 623 203 -517 -60 133 51 82 843 -262 297 -252 -356 14 -13 528 41 -660 -428 520 344 -901 831 -305 422 48 -250 -118 -647 -520 463 363 584 917 25 -499 917 -866 -705 -547 -547 200 -476 -324 -88 387 370 148 -237 128 -665 102 83 -757 -464 -318 -891 792 -541 -742 -300 -527 -272 540 -73 -703 -581 -425 553 364 -693 -883 693 728 805 -156 318 612 -798 641 108 -576 -492 244 -680 -81 232 -651 906 510 654 -407 580 218 -999 469 925 -53 -447 88 991 131 176 -317 122 -475 -725 941 -934 -985 -784 559 -821 -269 446 -699 -654 -569 413 488 36 559 453 -495 -101 -385 556 -662 500 -423 584 -503 -943 285 -625 -990 289 -762 523 486 -977 -491 -969 676 233 317 445 -906 -840 998 -745 398 -160 917 -186 -9 -561 -669 239 -784 540 350 -102 998 -201 -796 736 -432 802 -454 176 -447 -565 -424 71 82 510 754 -491 850 -755 63 -812 332 -78 175 253 330 807 93 961 -650 -943 57 -731 -517 942 783 -981 -18 -421 -638 -970 401 -509 -539 -428 -248 -742 923 -503 969 -769 540 -276 -626 -207 541 -135 923 303 529 -785 -156 988 -848 279 -157 -956 750 753 -508 -451 -179 -347 538 -694 -897 63 -798 864 -857 -786 17 -865 126 259 -232 578 143 695 173 -362 538 -855 -350 267 -644 8 397 -957 282 -594 -46 -651 -627 -362 -668 626 -947 -545 -437 197 -434 954 -897 -643 363 424 951 606 -422 -717 79 446 222 -4 665 61 -284 960 -565 969 882 130 625 -93 -152 456 -447 -959 -522 -896 555 -675 384 81 -665 -623 181 -123 -501 -94 -505 298 258 405 607 -143 -380 497 456 398 -141 934 670 369 156 -769 -769 451 -217 736 -837 -686 90 -620 -586 782 922 406 123 324 -684 687 832 -684 708 -703 869 -800 159 -87 464 -130 686 414 -382 -469 -631 -56 414 -21 636 -74 957 -939 -434 -503 -73 -362 -490 743 118 308 988 187 -532 -509 47 418 494 655 -50 -191 764 653 679 816 603 672 194 -873 104 989 511 -15 864 -989 -57 -109 955 -972 -507 -234 -708 -578 443 -9 -811 789 274 16 124 -4 -718 857 661 -542 448 -469 -603 369 606 -364 238 28 -802 -213 -624 558 -49 -909 730 147 493 632 -823 -182 539 -417 263 -618 -774 -576 -550 -528 -210 -678 610 293 -241 779 -736 -105 -10 753 -368 -728 549 -175 452 284 649 -454 45 274 -350 71 -774 -321 -307 -364 863 -14 -9 -794 412 754 -190 -933 147 946 -619 308 738 -734 398 881 -662 995 177 -322 413 890 713 -219 -629 654 600 -1 -490 577 -235 344 972 608 -365 286 -966 130 -706 519 497 534 293 999 727 802 -683 -970 137 -652 -261 392 -721 -477 714 -985 -128 -441 -914 325 746 920 356 311 -797 880 -841 -981 621 -185 -423 642 963 95 127 -644 673 -932 855 863 -192 372 225 -807 210 -535 -274 458 40 -170 -44 -28 -237 397 -157 -528 -930 463 -53 -494 632 598 -620 736 -465 127 298 420 -463 -132 -292 317 147 745 -646 842 -95 934 -99 725 -476 -225 -431 708 -742 745 206 412 -347 -603 -2 521 -407 897 -497 -544 -611 -206 -436 -886 -65 563 289 -405 -634 445 410 382 441 -397 813 954 237 142 58 119 -722 11 919 -762 -408 -702 -71 -143 203 -530 -798 -426 -553 826 487 218 600 -232 -455 120 438 -295 631 -701 -230 -257 -654 -1 897 842 597 629 187 -163 169 -226 -322 704 -724 -591 659 26 492 -663 -132 -565 -778 976 -803 -763 -549 301 -493 -200 197 -221 760 308 944 -35 490 -92 -336 -988 600 785 602 -372 448 543 113 177 -567 -63 -90 -381 -35 382 790 -337 -794 700 482 -173 817 750 450 -681 -14 349 -690 375 895 -812 -411 -798 737 -309 325 80 -994 933 703 365 -567 939 256 -326 -566 621 617 91 309 668 -249 -342 810 387 48 734 -56 404 748 -725 609 -770 -343 810 -526 88 287 -151 320 -737 645 -235 -147 566 806 162 -182 779 -205 184 700 674 991 802 -22 398 -566 190 103 -809 -822 405 -510 887 35 624 -280 553 57 0 -403 739 403 20 484 6 -637 -863 -186 663 649 -176 472 110 -702 -624 87 -991 428 -399 218 -53 537 -352 96 930 543 760 158 288 820 -427 -66 -803 129 799 833 -405 -796 327 266 769 430 -473 -976 752 -293 -801 92 -455 -210 404 153 7 132 885 246 51 -236 -819 472 494 679 -313 -285 178 56 268 -602 -594 -715 -297 494 673 -106 -405 -949 993 -756 171 -901 997 801 858 -436 -391 -695 -708 993 -787 401 -784 382 753 -51 -992 189 727 798 276 280 717 674 -669 -559 336 -927 379 509 71 3 -496 -299 -673 312 -35 -52 77 467 141 -911 -288 710 791 -201 -267 270 578 -216 -466 358 286 -41 71 338 -956 299 636 -220 723 -394 916 259 79 -221 -691 399 -821 -832 786 535 121 802 860 -147 -56 -856 619 -290 -407 -728 492 -349 -610 -143 334 -754 -285 868 999 930 -78 -723 -768 -435 -912 -451 572 103 -491 -224 621 -907 -735 -9 118 153 -716 -659 -10 446 357 -701 588 -772 -480 -226 -572 586 -400 -106 -305 151 539 464 -17 -291 372 621 -523 150 466 -828 -838 -903 151 565 135 -478 522 2 255 922 -230 -957 -618 -12 -976 312 -929 882 -197 699 744 -133 -514 738 -441 -204 -146 -284 368 -814 484 325 541 -531 457 580 -293 796 -306 966 -321 287 -441 242 -62 -532 -563 -811 -699 -307 -189 953 -430 -667 563 48 663 700 610 -607 -779 55 -199 -314 451 -648 -590 320 -362 627 589 -942 27 492 -790 915 -320 -485 -320 -210 -332 -908 -734 -323 322 -2 -863 -978 460 389 988 -606 -813 -143 422 -243 671 -957 -869 896 -30 105 -737 778 -570 105 -157 -416 -873 -333 -943 -685 -587 -349 114 -113 996 464 973 -800 -222 566 -121 449 -677 808 502 176 457 -82 998 341 -295 -288 736 -99 304 -614 -111 105 -120 -230 -49 -724 -680 910 457 780 717 -767 -69 98 -959 840 0 -921 -659 321 -249 866 913 -264 -298 -997 -79 -311 -186 751 892 -284 67 727 -541 -407 -333 -504 -372 24 -505 -626 524 7 -943 -377 -62 969 -465 188 -967 874 -781 -608 -760 -822 -703 130 -917 694 605 -128 454 -369 -289 -337 -1 249 643 512 -341 738 156 -341 913 -894 -540 -988 307 867 740 -268 -658 275 -141 618 869 764 -535 -387 -961 342 595 541 654 903 194 -447 -514 218 907 -924 886 948 153 997 -228 -758 -418 436 319 293 434 -97 -171 717 214 728 -960 946 -729 497 -979 541 -139 -919 -845 838 -348 -114 714 700 741 229 684 702 -392 -280 198 77 -987 71 873 856 770 689 -794 -7 42 -155 673 840 -142 106 892 -430 -373 -804 -226 -276 -633 848 81 462 842 -834 -32 -201 -618 -264 -629 -182 575 863 -593 -83 -588 -214 503 569 -743 961 193 -303 172 712 235 -62 -281 358 381 635 617 475 245 352 289 -762 843 188 -494 636 788 -782 588 -467 -255 -350 -880 -687 819 -980 128 -365 -927 -806 677 426 392 806 232 -428 -975 18 -946 970 -224 371 314 656 412 -863 228 468 712 743 588 -83 20 185 255 45 -717 894 314 196 827 973 824 281 667 -951 209 639 -894 -841 -654 694 -113 134 230 -537 -187 506 252 -449 666 8 -518 -600 -127 -964 399 302 485 889 -343 904 765 277 -702 -387 445 -400 348 -70 36 -209 156 854 928 641 268 -47 929 782 -774 113 227 905 -170 -984 968 -448 459 -628 563 389 -333 319 -230 805 -245 242 29 441 865 880 521 -305 252 -41 -302 -276 -508 441 -317 505 227 454 122 -682 721 -126 -319 548 -888 806 -776 200 -94 -427 -325 -796 775 -261 803 -211 897 -152 11 697 358 2 -713 -272 -646 127 415 574 586 -8 -927 -900 -962 991 805 -720 -296 4 -795 284 514 -524 -770 877 228 405 71 997 -427 507 -628 337 -739 558 795 -244 -530 130 -367 266 -979 461 371 270 -605 -932 -200 412 -137 239 -444 -912 863 824 437 -749 458 552 -218 -935 55 795 -35 -562 -578 687 796 -192 -857 -88 5 783 687 -887 -29 -656 442 454 898 -849 -557 678 163 276 -263 929 815 38 -867 736 -779 81 70 -407 584 554 675 -748 346 -500 -242 -259 -474 403 -69 741 165 770 650 -676 907 406 -476 105 241 -184 289 518 -7 215 541 836 -906 -803 -534 423 -475 883 -911 717 695 -633 -613 50 -69 25 405 -559 819 -941 -20 -334 -385 -379 -36 -688 884 597 294 -192 501 -368 -187 813 -904 390 -737 235 -525 -269 -85 -418 474 -741 -819 -718 470 -462 823 566 488 112 496 -781 574 145 -695 -444 151 786 -316 -165 401 506 354 -849 -155 513 510 -924 701 -278 380 -596 163 -59 -323 670 742 -148 300 -296 515 -915 870 -752 865 694 -60 -307 479 943 -13 735 -196 -715 516 -906 