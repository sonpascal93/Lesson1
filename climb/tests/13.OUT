192
73 89 93 111 112 138 149 186 195 203 210 268 360 388 397 507 510 512 543 586 629 710 739 754 780 783 811 835 878 886 911 921 928 950 958 964 965 971 990 1003 1025 1038 1043 1056 1062 1095 1107 1175 1199 1232 1245 1369 1376 1378 1409 1547 1554 1585 1601 1646 1662 1719 1726 1740 1767 1847 1903 1927 1996 2025 2033 2116 2148 2188 2232 2249 2261 2263 2282 2299 2308 2314 2317 2333 2390 2449 2502 2506 2520 2577 2578 2580 2613 2655 2661 2662 2718 2723 2747 2809 2813 2830 2889 2902 2972 2978 2993 3049 3070 3127 3134 3140 3141 3150 3154 3184 3219 3226 3277 3319 3379 3398 3408 3413 3415 3426 3433 3438 3448 3470 3474 3507 3513 3533 3559 3596 3614 3628 3660 3666 3699 3701 3742 3761 3850 3858 3876 3878 3923 3949 3999 4056 4058 4130 4208 4218 4224 4260 4276 4303 4320 4329 4331 4342 4356 4369 4371 4373 4402 4467 4496 4501 4504 4559 4638 4644 4712 4714 4734 4755 4804 4819 4846 4847 4858 4892 4928 4939 4946 4947 4968 4972 